example/contracts_generator.yaml

```diff
@@ -1,5 +1,20 @@
-# Path to contracts' .csproj project file (required)
-input: ../contractsgenerator/examples/project/aggregated/Combined/Combined.csproj
+# Possible input methods, specify exactly one (required)
+input:
+  # Array of paths pointing to .csproj contracts project files
+  # project:
+  #   - ../contractsgenerator/examples/project/aggregated/Combined/Combined.csproj
+  # Path to a single source file
+  # file: ExampleContracts/Auth.cs
+  # Multiple files selected by glob patterns
+  path:
```

Comment:
Try to favour `project` source - it will be slightly slower in generation, but will be easier to manage (if someone decides to split the project into smaller ones, e.g. `common` + `core`).